### ISSUE TITLE
gene splitted into two contigs?

### ISSUE THREAD

**Issue by elina2410** [tseemann/abricate#202]
Hello, I need help in interpreting abricate (vfdb) result.
Based on result below can I assume that there is hilA gene in the analyzed strain? I mean if I can sum coverage results for these fragments and assume that I have 100% coverage for hilA gene? 

![obraz](https://github.com/tseemann/abricate/assets/105778536/550328ca-0777-4bbd-9ff3-99b5859ebe1e)

How would you interpret these results?

Thank you in advance.

**Comment by ammaraziz** [comment]
Yes that's correct, you have hilA gene. Looks like during denovo assembly (spades?) the gene was split in half across two contigs. 

Post denovo you can perform scaffolding ( I like ragtag) which should merge the two contigs. 

Assumptions are that you have only one copy of hilA, this is an isolate with one strain and you have correctly performed denovo.

**Comment by tseemann** [comment]
@ammaraziz is correct.

Not clear why the assembly broke in the middle.
Could be low coverage due to extreme GC content.

I would try re-assembling with Spades `--isolate` mode.
